### DISCUSSION TITLE
install openjdk 11 or 8

### DISCUSSION THREAD

**Discussion by ganeshkamal-ar** [termux/termux-app#2341] [Q&A] [answered]
How to install openjdk 11 or 8

**Comment by hnmn** [comment, marked as answer]
Termux official have only openjdk-17 available 
But there is openjdk 11 available here [https://github.com/Lzhiyong/termux-ndk/releases/download/openjdk/openjdk-11.0.12.tar.xz](https://github.com/Lzhiyong/termux-ndk/releases/download/openjdk/openjdk-11.0.12.tar.xz)

**Reply by ganeshkamal-ar** [reply]
How to install it

**Reply by hnmn** [reply]
`tar xvf path-to-openjdk `
Follow instructions at 
https://github.com/Lzhiyong/termux-ndk/releases/tag/openjdk
